src/service/standard-actions-impl.js

```diff
@@ -179,6 +192,60 @@ export class StandardActions {
   }
 
   /**
+   * Handles the `scrollTo` action where given an element, we smooth scroll to
+   * it with the given animation duraiton
+   * @param {!./action-impl.ActionInvocation} invocation
+   */
+  handleScrollTo(invocation) {
+    if (!invocation.satisfiesTrust(ActionTrust.MEDIUM)) {
+      return;
+    }
+    const node = dev().assertElement(invocation.target);
+
+    // Duration for scroll animation
+    const duration = invocation.args
+                     && invocation.args['duration']
+                     && invocation.args['duration'] >= 0 ?
+                     invocation.args['duration'] : 500;
```

Comment:
please create an issue to make the default be based on some formula that account for the distance between source and target. Ideally whatever Chrome is using for their native implementation (https://bugs.chromium.org/p/chromium/issues/detail?id=243871)